modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -55,7 +55,7 @@ class AutoEstablishedClaim < ApplicationRecord
 
     def to_internal
       form_data['applicationExpirationDate'] ||= build_application_expiration
-      form_data['claimDate'] ||= (persisted? ? created_at.to_date.to_s : Time.zone.today.to_s)
+      form_data['claimDate'] ||= (persisted? ? created_at.iso8601 : Time.zone.today.to_s)
```

Comment:
for consistency, would it make sense for that same formatting to be applied to the latter part of the ternary? i.e. right now they're different:
```ruby
DateTime.parse(Time.now.to_s).iso8601
# => "2022-06-01T20:03:33-05:00"
Time.zone.today.to_s
# => "2022-06-02"

# or we could do this instead to make the formats match
Time.zone.now.iso8601
# => "2022-06-02T01:12:20Z"
```